src/app/pages/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
The Next.js App Router uses the app folder for routing, so where the file is located is important for routing. The filepath for this file should be src/app/reset-password. That way, when someone goes to the /reset-password endpoint, this page will load. Also, the file should be named page.tsx.